data/json/recipes/recipe_food.json

```
@@ -1913,6 +1913,21 @@
     "tools": [ [ [ "surface_heat", 3, "LIST" ] ] ],
     "components": [ [ [ "potato", 1 ], [ "irradiated_potato", 1 ] ] ]
   },
+  {
+    "type": "recipe",
+    "activity_level": "NO_EXERCISE",
+    "result": "potato_boiled",
+    "category": "CC_FOOD",
+    "subcategory": "CSC_FOOD_VEGGI",
+    "skill_used": "cooking",
+    "difficulty": 1,
+    "time": "15 m",
+    "batch_time_factors": [ 80, 4 ],
+    "autolearn": true,
+    "qualities": [ { "id": "BOIL", "level": 1 } ],
+    "tools": [ [ [ "surface_heat", 2, "LIST" ] ] ],
+    "components": [ [ [ "potato", 1 ], [ "water", 1 ], [ "water_clean", 1 ] ] ]
```

May want to allow using irradiated potatoes here as well. Also this syntax incorrectly assumes that you can substitute potato with water.
```suggestion
    "components": [ [ [ "potato", 1 ], [ "irradiated_potato", 1 ] ], [ [ "water", 1 ], [ "water_clean", 1 ] ] ]
```